hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/TarContainerPacker.java

```diff
@@ -114,12 +116,11 @@
     }
   }
 
+  @SuppressFBWarnings("NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE")
```

Comment:
Can we pls add minimal javadoc to explain why we are suppressing this warning? This will ensure that in future anyone changing this code path, is cautious.

Reply:
Thanks @dineshchitlangia for the review.  I agree about the need for explanation, I was just kind of lazy adding it.  In fact, I think it's simpler to replace the suppression with a null check. :)